scipy/ndimage/_interpolation.py

```diff
@@ -654,6 +654,117 @@ def shift(input, shift, output=None, order=3, mode='constant', cval=0.0,
     .. versionadded:: 1.6.0
         Complex-valued support added.
 
+    Examples
+    --------
+    A standard call to `shift` will shift ``input`` values by the specified
+    ``shift`` number of indices in along all dimensions. Fractional values
```

Comment:
I find the use of "indices" here and elsewhere confusing because it doesn't appear elsewhere in the docs so is not immediately obvious what it refers to